src/components/profile/orders/index.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { useState } from "react";
+import styled from "styled-components";
+import FoodCard from "../../common/FoodCard";
+import SearchIcon from "../../../assets/restaurantDetails/icon_search.png";
+import data from "../../common/constants.json";
+import { fontFamilies, colors, sizes, opacity } from "../../../variables";
+import { orderCardProps } from "../../common/interfaces";
+import OrderCard from "../../common/orderCard";
+import Dropdown from "../../common/Dropdown";
+
+const Orders = () => {
+  return (
+    <Wrapper>
+      <Dropdown
+        name="OrderDropdown"
+        className="semibold"
+        arrow={true}
+        options={[
```

Comment:
Can you use map here ?

Reply:
Instead of mapping i moved those data to constants.json file